spec/factories/dishes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :dish do
+    name "MyString"
```

Comment:
This is generic name.- Maybe change to real value?

Reply:
Done